window_app.py

```diff
@@ -1,17 +1,26 @@
-from PyQt5.QtWidgets import QApplication, QWidget, QFileDialog, QPushButton, QLabel, QVBoxLayout
+from PyQt5.QtWidgets import QApplication, QWidget, QFileDialog, QPushButton, QLabel, QVBoxLayout, QShortcut
+from PyQt5.QtGui import QKeySequence
 from PyQt5.QtCore import pyqtSlot, Qt
 import sys
 
+exit_shortcut_conf = "Ctrl+Q"

```

Comment:
Tutaj można wydzielić zmienne odpowiedzialne za ustawienia aplikacji (których wraz z rozwojem projektu będzie coraz to więcej) do osobnego pliku. Możecie zrobić json (choć będzie to trochę niepraktyczne), własny format (ugh!) lub napisać prosty osobny plik z rozszerzeniem .py, importować go i czytać z niego zmienne. Możecie też skorzystać z biblioteki "configparser". 
Fajny wątek na temat ustawień aplikacji znajduje się tutaj:
https://stackoverflow.com/questions/6198372/most-pythonic-way-to-provide-global-configuration-variables-in-config-py